docs/content/how-do-i-write-policies.md

```diff
@@ -1053,6 +1053,79 @@ The result:
 +-----------+------------------------+
 ```
 
+## Universal Quantification (FOR ALL)
+
+Like SQL, Rego does not have a direct way to express _universal quantification_
+("FOR ALL"). However, like SQL, you can use other language primitives (e.g.,
+[Negation](#negation)) to express FOR ALL. For example, imagine you want to
+express a policy that says (in English):
+
+```
+There must be no apps named "bitcoin-miner".
+```
+
+You may start by defining a rule named `no_bitcoin_miners` like so:
+
+```ruby
+no_bitcoin_miners {
+    app := apps[_]
+    app.name != "bitcoin-miner"  # THIS IS NOT CORRECT.
+}
+```
+
+Unfortunately this rule is incorrect. You can see this by testing the rule:
```

Comment:
Might also use `some` to show that this is incorrect....

"It becomes clear that this is incorrect when you use the `some` keyword, because the rule is true whenever there is some app that is not a bitcoin-miner."
````
no_bitcoin_miners {
    some i
    app := apps[i]
    app.name != "bitcoin-miner"
}
```